libc/test/src/__support/FPUtil/fpbits_test.cpp

```diff
@@ -425,13 +425,10 @@ TEST(LlvmLibcFPBitsTest, DoubleType) {
   EXPECT_EQ(quiet_nan.is_quiet_nan(), true);
 }
 
-#ifdef LIBC_TARGET_ARCH_IS_X86
+#ifdef LIBC_TYPES_LONG_DOUBLE_IS_X86_FLOAT80
```

Comment:
Should probably explicitly `#include "libc/src/__support/macros/properties/types.h"`.

Reply:
Updated.